api/pb/api.proto

```diff
@@ -135,4 +157,8 @@ service API {
     rpc GetProject (GetProjectRequest) returns (GetProjectReply) {}
     rpc ListProjects (ListProjectsRequest) returns (ListProjectsReply) {}
     rpc RemoveProject (RemoveProjectRequest) returns (RemoveProjectReply) {}
+
+    rpc AddAppToken (AddAppTokenRequest) returns (AddAppTokenReply) {}
+    rpc ListAppTokens (ListAppTokensRequest) returns (ListAppTokensReply) {}
+    rpc RemoveAppToken (RemoveAppTokenRequest) returns (RemoveAppTokenReply) {}
```

Comment:
This looks good for now, but just fyi... I started building grpc components in different packages in the `filecoin` project. Feels pretty nice and keeps the server and client and proto code from getting monolithic. Maybe worth considering at some point.

Reply:
Cool, makes sense. Might make more sense over there where there's a clean break between modules.